stablehlo/dialect/TypeInference.cpp

```diff
@@ -3366,6 +3368,80 @@ LogicalResult verifyCollectivePermuteOp(
   return success();
 }
 
+LogicalResult verifyCompositeOp(std::optional<Location> loc, Operation* op,
+                                StringRef name, StringRef decomposition) {
+  // composite_c1
+  llvm::SmallVector<llvm::StringRef> parts;
+  name.split(parts, '.');
```

Comment:
you could probably use the split that returns a pair since all you want is at most 1 split.

Reply:
We actually support more than 1 split though.